kitchen-tests/kitchen.travis.yml

```diff
@@ -104,6 +104,9 @@ platforms:
   driver:
     image: dokken/opensuse-leap
     pid_one_command: /bin/systemd
+    intermediate_instructions:
+      - RUN /usr/bin/zypper --non-interactive update
+      - RUN /usr/bin/zypper --non-interactive install cron
```

Comment:
if we need cron we should just add it to the dokken-images, but I can do this later. It's not a blocker on this.